### DISCUSSION TITLE
Insert an item into row on load

### DISCUSSION THREAD

**Discussion by blessenabrahamnw18** [komarovalexander/ka-table#312] [Q&A]
Hi @komarovalexander 
I need to insert rowkeyfield once the data is fetched from the backend API.
Is it possible without me doing a normal map/loop and insert the value into each row.
Ka-table should be doing a loop which can be used to insert this value?

**Comment by komarovalexander** [comment]
Hi @blessenabrahamnw18 you can use insertRow action to insert the row https://ka-table.com/docs_action.html#Action

Or If you you need to insert value into existing row, I would suggest to use UpdateCellValue then
